### DISCUSSION TITLE
Spelling and Grammar Tools

### DISCUSSION THREAD

**Discussion by dougransom** [James-Yu/LaTeX-Workshop#3504] [Q&A] [answered]
What are suggested grammar and spelling tools that work with Visual Studio Code and/or Latex-Workshop?  I am on Windows and use Canadian/British spellings of words like 'colour'.

**Comment by PierreMarchand20** [comment, marked as answer]
I like [ltex](https://marketplace.visualstudio.com/items?itemName=valentjn.vscode-ltex), it has spelling and grammar tools for various languages. It uses [language tool](https://languagetool.org) and it has support for latex syntax.
